src/test/java/com/salesforce/storm/spout/dynamic/kafka/ConsumerTest.java

```diff
@@ -454,8 +461,10 @@ public void testConnectWithSinglePartitionOnTopicWithNoStateSaved() {
         // Since ConsumerStateManager has no state for partition 0, we should call seekToBeginning on that partition
         verify(mockKafkaConsumer, times(1)).seekToBeginning(eq(Lists.newArrayList(kafkaTopicPartition0)));
 
-        // Verify position was asked for
-        verify(mockKafkaConsumer, times(1)).position(kafkaTopicPartition0);
+        final ConsumerRecords<byte[], byte[]> consumerRecords = makeConsumerRecords(0, 999);
```

Comment:
@Crim Setting the offset here to 999 makes the test past but also fundamentally changes the nature of the test.  I think we need to agree on a direction here before I fix the remaining test failures.